packages/zcli-themes/src/commands/themes/publish.ts

```diff
@@ -0,0 +1,43 @@
+import { Command, Flags, CliUx } from '@oclif/core'
+import { request } from '@zendesk/zcli-core'
+import * as chalk from 'chalk'
+
+export default class Publish extends Command {
+  static description = 'publish a theme'
+
+  static flags = {
+    themeId: Flags.string({ description: 'The id of the theme to publish' })
+  }
+
+  static args = [
+    { name: 'themeDirectory', required: true, default: '.' }
```

Comment:
Wait, it seems like we do accept a `themeDirectory`, we just need to document it? :)

Reply:
Good catch! I removed it entirely as it's not needed 👍🏼